src/warc2zim/content_rewriting/css.py

```diff
@@ -23,14 +36,26 @@ def rewrite(self, content: str | bytes) -> str:
             rules = parse_stylesheet(content)
         self.process_list(rules)
 
-        output = serialize(rules)
+        try:
+            output = serialize(rules)
+        except Exception:
+            fallback_rewriter = FallbackRegexCssRewriter(self.url_rewriter)
```

Comment:
I would prefer to initialize the `fallback_rewriter` in the `__init__`, it feels weird to have two places where we initialize this object (inline and not inline) and potentially two different ways to do it. WDYT?